tools/sst_dump_test.cc

```diff
@@ -286,6 +286,25 @@ TEST_F(SSTDumpToolTest, ReadaheadSize) {
   }
 }
 
+TEST_F(SSTDumpToolTest, NoSstFile) {
+  Options opts;
+  opts.env = env();
+  std::string file_path = MakeFilePath("no_such_file.sst");
+  char* usage[3];
+  PopulateCommandArgs(file_path, "--show_properties", usage);
+  ROCKSDB_NAMESPACE::SSTDumpTool tool;
+  for (const auto& command :
+       {"--command=check", "--command=dump", "--command=raw",
+        "--command=verify", "--command=recompress",
+        "--command=verify_checksum"}) {
+    snprintf(usage[1], kOptLength, "%s", command);
```

Comment:
Doesn't this overwrite "--show_properties"? Maybe we don't need it, but we should at least start with something like empty string to make it less confusing.